WireAuthentication/Sources/WireAuthenticationUI/Views/DetermineAuthMethod/DetermineAuthMethodView.swift

```diff
@@ -108,6 +108,13 @@ package struct DetermineAuthMethodView: View {
                 loginViaEmailBuilder.loginViaEmailView(email: email, canCreateAccount: false)
             case .loginOrRegister:
                 Color.red
+            case .noHistory:
+                Color.blue
+            }
+        }
+        .onReceive(NotificationCenter.default.publisher(for: .ssoLoginDidFinishNotification)) { _ in
+            Task {
+                viewModel.dismissmodalView()
```

Comment:
question: is this notification really necessary, given that on SSO success we completely replace `DetermineAuthMethodView`?

Reply:
No, it's not needed. Removed